README.md

```diff
@@ -96,4 +97,10 @@ By default, the underlying driver is set to error level logging. It can be chang
 
 ## Contributing
 
-Cf. [Contributing](./CONTRIBUTING.md).
+Check [Contributing](./CONTRIBUTING.md).
+
+## Releases
+
+Releases will be performed as needed, typically once every 2 weeks.
+
+Releases are published to [the terraform registry](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest). Each change has its own release notes (e.g. https://github.com/Snowflake-Labs/terraform-provider-snowflake/releases/tag/v0.89.0) and migration guide if needed (e.g. https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/MIGRATION_GUIDE.md#v0880--v0890).
```

Comment:
Links shouldn't point to specific versions which will be quickly obsolete. E.g. https://github.com/Snowflake-Labs/terraform-provider-snowflake/releases, https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/MIGRATION_GUIDE.md

Reply:
Why not? This are just examples, and we won't remove either of them.